README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
-  executor[/"Compute Service"\] --> del2Aud
-  rootIss --> owner
-
-  executor -. accesses .-> resource
-  rootCap -. references .-> resource
-
-  subgraph root [Root UCAN]
-    rootIss(iss: Alice)
-    rootAud(aud: Bob)
-    rootCap("cap: (Storage, crud/*)")
-  end
+This field SHOULD NOT be used to sign arbitrary data, such as signature challenges. See the [`meta`][Metadata] field for more.
 
-  subgraph del1 [Delegated UCAN]
-    del1Iss(iss: Bob) --> rootAud
-    del1Aud(aud: Carol)
-    del1Cap("cap: (Storage, crud/*)") --> rootCap
-  end
+Here is a simple example.
 
-  subgraph del2 [Final UCAN]
-    del2Iss(iss: Carol) --> del1Aud
-    del2Aud(aud: Compute Service)
-    del2Cap("cap: (Storage, crud/*)") --> del1Cap
-  end
+``` js
+{
+  // ...
+  "nonce": {"/": {"bytes": "bGlnaHQgd29yay4"}}
+}
 ```
 
-In the above diagram, Alice has some storage. This storage may exist in one location with a single source of truth, but to help build intuition this example is location independent: local versions and remote stored copies are eventually consistent, and there is no one "correct" copy. As such, we list the owner (Alice) directly on the resource.
+# Metadata
 
-Alice delegates access to Bob. Bob then redelegates to Carol. Carol invokes the UCAN as part of a REST request to a compute service. To do this, she MUST both provide proof that she has access (the UCAN chain), and MUST delegate access to the discharging compute service. The discharging service MUST check that the root issuer (Alice) is in fact the owner (typically the creator) of the resource. This MAY be listed directly on the resource, as it is here. Once the UCAN chain and root ownership are validated, the storage service performs the write.
+The OPTIONAL `meta` field contains a map of arbitrary metadata, facts, and proofs of knowledge. The enclosed data MUST be self-evident and externally verifiable. It MAY include information such as hash preimages, server challenges, a Merkle proof, dictionary data, etc.
 
-### 6.2.1 Recipient Validation
+The data contained in this map MUST NOT be semantically meaningful to delegation chains.
 
-An agent executing a capability MUST verify that the outermost `aud` field _matches its own DID._ The associated ability MUST NOT be performed if they do not match. Recipient validation is REQUIRED to prevent the misuse of UCANs in an unintended context.
-
-The following UCAN fragment would be valid to invoke as `did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV`. Any other agent MUST NOT accept this UCAN. For example, `did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp` MUST NOT run the ability associated with that capability.
+Below is an example:
 
 ``` js
 {
-  "aud": "did:key:zH3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV",
-  "iss": "did:key:zAKJP3f7BD6W4iWEQ9jwndVTCBq8ua2Utt8EEjJ6Vxsf",
   // ...
+  "meta": {
+    "challenges": {
+      "example.com": "abcdef",
+      "another.example.net": "12345"
+    },
+    "sha3_256": {
```

Comment:
not `SHA-256`, i.e. SHA2? or is that deliberate?

Reply:
This is an example of putting in arbitrary data. It literally doesn't matter